dao/src/main/kotlin/tables/OrtRunsTable.kt

```diff
@@ -114,4 +116,36 @@ class OrtRunDao(id: EntityID<Long>) : LongEntity(id) {
         traceId = traceId,
         environmentConfigPath = environmentConfigPath
     )
+
+    /**
+     * Map this [OrtRunDao] to an [OrtRunSummary] which is a reduced representation of an [OrtRun].
+     * Reduce the load on the database by utilizing lazy loading (only actually required fields are loaded).
```

Comment:
This comment about lazy loading confuses me, since I somehow was expecting clever tweaks on the Exposed configuration to change the loading behavior of associations or whatever. But in fact, you just access a subset of the entity's properties. So, could you just state something like "This prevents that unnecessary data is loaded from the database." ?

Reply:
Yes, there is nothing that you need to actively do to make use of lazy loading, you just must not make the mistake to kind of "touch" attributes that would cause Exposed to load related data from other database tables.